src/shared/copy.h

```diff
@@ -40,6 +40,7 @@ typedef enum CopyFlags {
          * copy because reflinking from COW to NOCOW files is not supported.
          */
         COPY_NOCOW_AFTER                  = 1 << 20,
+        COPY_FS_VERITY                    = 1 << 21, /* Preserve fs-verity when copying. */
```

Comment:
maybe name this COPY_PRESERVE_FS_VERITY right away. i sense sooner or later we might end up with another flag COPY_ADD_FS_VERITY or so, hence better prepare for thta and make clear what this is righ tfrom the beginning

Reply:
but it would be per-copy-operation, not per-fs

my understanding was that ostree for example enables fs-verity for a whole directory subtree, no? or does it mix fs-verity and non-fs-verity files within the same dirs?

Reply:
No: that's true.  The entire /composefs/objects/ directory gets fs-verity enabled for every file.  But: many other things in the repository (ie: other files in `/composefs/`) won't have it enabled, and it's kinda weird to copy the repository in two separate pieces...